docs/judge/linux_installation.md

```diff
@@ -3,7 +3,7 @@ This installation guide is for Linux based machines (WSL included). For installa
 ## Installing the prerequisites
 
 ```
-$ apt install git python-dev python-pip build-essential
+$ apt install git python-dev python-pip build-essential libseccomp-dev
```

Comment:
The [README.md](https://github.com/dmoj/judge#installation) in the judge goes into a lot more detail on this. `libseccomp-dev` is only _required_ if you want to build with seccomp enabled. If you pass in `DMOJ_USE_SECCOMP=no` to the build (e.g., because the kernel you are building on does not support seccomp), it will build just fine. 

That said, this addition will save some people time even without that distinction.